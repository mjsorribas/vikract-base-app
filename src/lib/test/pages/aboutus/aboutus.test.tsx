import { MemoryRouter } from "react-router-dom";
import renderer from "react-test-renderer";
import { expect, test } from "vitest";

import Aboutus from "lib/pages/aboutus/index";

const toJson = (component: renderer.ReactTestRenderer) => {
  const result = component.toJSON();
  expect(result).toBeDefined();
  return result as renderer.ReactTestRendererJSON;
};

test("Contact Us Page", () => {
  const component = renderer.create(
    <MemoryRouter initialEntries={[{ pathname: "/" }]}>
      <Aboutus />
    </MemoryRouter>
  );
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
