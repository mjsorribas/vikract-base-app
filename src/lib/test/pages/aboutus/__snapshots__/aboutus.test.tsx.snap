// Vitest Snapshot v1

exports[`About Us Page 1`] = `
<div
  className="css-qi60vh"
>
  <h2
    className="chakra-heading css-0"
  >
    What is an About Us page?
  </h2>
  <p
    className="chakra-text css-0"
  >
    An About Us page is a spot for your founding story, a place to show off your business wins, and a sales page that answers the most pressing question new customers have about your business.
    <br />
    What an About Us page should be is a goal-oriented sales page, one that focuses on highlighting the biggest selling points of your story and brand at the top of the page, making a strong first impression on curious customers.
  </p>
</div>
`;
